src/napari_imagej/_function.py

```diff
@@ -399,13 +529,15 @@ def __init__(self, napari_viewer: Viewer):
 
         self.layout().addWidget(self._search_widget)
 
-        self.searcher: QLineEdit = self._generate_searcher()
+        self.searchers, self.resultConverters = self._generate_searchers()
```

Comment:
Returning two things from one function kinda  sucks.

Can we try to find a way around that?

Reply:
Why does it suck? Python is good at doing it, and it's something the right thing.

Reply:
Sure, but it doesn't quite feel like it in this case.

The things it returns are only remotely related.